xsd/widget.xsd

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
this is not the latest version, please update. [https://github.com/mendix/AppStoreWidgetBoilerplate/blob/master/xsd/widget.xsd](https://github.com/mendix/AppStoreWidgetBoilerplate/blob/master/xsd/widget.xsd)